### PR TITLE
cd, free, neofetch, reboot: add Bengali translation

### PR DESCRIPTION
**Pull request by sandy3002**
Added:
#free
#reboot
#cd
#neofetch

- [ ✅  ] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [ ✅ ] The page(s) have at most 8 examples.
- [ ✅ ] The page description(s) have links to documentation or a homepage.
- [ ✅ ] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [ ✅ ] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages.bn/common/cd.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# cd
+
+> বর্তমান কাজের ডিরেক্টরি পরিবর্তন করুন।
+> আরো তথ্যের জন্য: <https://manned.org/cd>।
+
+- প্রদত্ত ডিরেক্টরিতে যান:
+
+`cd {{ডিরেক্টরি / এর / পথ}}`
```
Comment (navarroaxel) [inline pages.bn/common/cd.md:8]:
```suggestion
`cd {{ডিরেক্টরি/এর/পথ}}`
```

File: pages.bn/linux/free.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# free
+
+> সিস্টেমে `Free` এবং ব্যবহৃত মেমোরির মান দেখায়
+> আরো তথ্যের জন্য: <https://manned.org/free>.
```
Comment (navarroaxel) [inline pages.bn/linux/free.md:4]:
```suggestion
> আরও তথ্য পাবেন: <https://manned.org/free>.
```

File: pages.bn/linux/reboot.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
+# reboot
+
+> মেশিনকে `reboot` করে
+> আরো তথ্যের জন্য: <https://manned.org/reboot.8>.
```
Comment (navarroaxel) [inline pages.bn/linux/reboot.md:4]:
```suggestion
> আরও তথ্য পাবেন: <https://manned.org/reboot.8>.
```

File: pages.bn/linux/free.md (hunk 2 above)
Comment (sandy3002) [reply pages.bn/linux/free.md:4]: আরো is more natural than আরও. If you still want the change, I will correct this

File: pages.bn/common/cd.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
+# cd
```
Comment (kbdharun) [inline pages.bn/common/cd.md:1]: This transaction isn't in sync with the [main page](https://github.com/tldr-pages/tldr/blob/main/pages/common/cd.md). Add the missing command and reorder it in correct order.

File: pages.bn/linux/reboot.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+
+- সিস্টেমকে থামিয়ে দেয়:
+
+`reboot --halt`
+
+- Sysadmin এর সাথে যোগাযোগ না করেই তৎক্ষণাৎ রিবুট করে:
+
+`reboot --force`
+
+- সিস্টেমকে রিবুট না করেই wtmp শাটডাউন এন্ট্রি করে:
```
Comment (kbdharun) [inline pages.bn/linux/reboot.md:22]:
```suggestion
- সিস্টেমকে রিবুট না করেই `wtmp` শাটডাউন এন্ট্রি করে:
```

**Review by kbdharun — CHANGES_REQUESTED** [review]

**Pull request closed without merging**